lib/js/jslib.js

```diff
@@ -1404,9 +1404,66 @@ const LINKS = new (function() {
 const _debug = DEBUG.debug;
 const debug = LINKS.kify(DEBUG.debug);
 
+function _regexCompile(expr){ 
+  switch(expr._label){
+    case "Seq": {
+      const regexes = LINKEDLIST.toArray(expr._value);
+      let acc = "";
+      for(let regex of regexes){
+        acc = acc + _regexCompile(regex);
+      }    
+      return acc;
+    }
+    case "StartAnchor":{
+      return "^";
+    }
+    case "Repeat":{
+      let op = undefined;
+      switch(expr._value[1]._label){
+        case "Plus": op = "+"; break;
+        case "Star": op = "*"; break;
+        case "Question": op = "?"; break;
+        default: throw("Error: Unrecognised Operation");
```

Comment:
Make the error message a little bit more descriptive, e.g. `"Unrecognised operation '" + expr._value[1]._label + "'"`. Perhaps use `_error` rather than `throw`.

Reply:
Fixed.